### ISSUE TITLE
Text nodes in code blocks are escaped

### ISSUE THREAD

**Issue by matthewp** [canjs/can-simple-dom#29]
`<anonymous>` becomes `&lt;anonymous&gt;`

**Comment by matthewp** [comment]
Going to first verify i'm write about this in a browser.

**Comment by matthewp** [comment]
``` html
<body>
<script>
  (function(){
    var code = document.createElement("code");
    code.appendChild(document.createTextNode("<anonymous>"));
    document.body.appendChild(code);
  })();
</script>
</body>
```

**Comment by justinbmeyer** [comment]
@matthewp is this possible to land?

**Comment by christopherjbaker** [comment]
Is this fixed with #30?

**Comment by matthewp** [comment]
I don't remember the context for this bug, so I don't know. Feel free to give it a shot and see if it works for you.
